test/integration/esClient.js

```diff
@@ -0,0 +1,64 @@
+const chai = require('chai');
+const esClient = require('../../app/lib/esClient');
+const utils = require('./testUtils');
+
+const expect = chai.expect;
+
+describe('esClient', function test() {
+  this.timeout(utils.maxWaitTimeMs);
+
+  before((done) => {
+    utils.waitForSiteReady(done);
+  });
+
+  describe('getQuery', () => {
+    const location = { longitude: -1.46519099452929, latitude: 54.0095586395326 };
+    it('should return pharmacies as an array', (done) => {
+      esClient.getPharmacies(location).then((pharmacies) => {
+        expect(pharmacies).to.be.an('array');
+        expect(pharmacies.length).to.be.greaterThan(0);
+        expect(pharmacies[0].name).to.exist;
+        done();
+      }).catch(done);
+    });
+
+    it('number of results should default to 2500', (done) => {
+      esClient.getPharmacies(location).then((pharmacies) => {
+        expect(pharmacies).to.be.an('array');
+        expect(pharmacies.length).to.be.lessThan(2501);
+        expect(pharmacies.length).to.equal(529);
+        done();
+      }).catch(done);
+    });
+
+    it('number of results should be parameterised', (done) => {
+      esClient.getPharmacies(location, 10, 7).then((pharmacies) => {
+        expect(pharmacies).to.be.an('array');
+        expect(pharmacies.length).to.equal(7);
+        done();
+      }).catch(done);
+    });
+
+    it('should return pharmacies populated with distance from location', (done) => {
```

Comment:
Possibly another one for inspecting the generated query?
If not, extending the expectations to check the distance increases each time could be useful as this would test the query has been correctly created for adding the sort criteria.